src/middleware/packages/triplestore/service.js

```diff
@@ -19,12 +19,13 @@ const TripleStoreService = {
     user: null,
     password: null,
     mainDataset: null,
+    fusekiBase: null,
     // Sub-services customization
     dataset: {}
   },
   dependencies: ['jsonld'],
   async created() {
-    const { url, user, password, dataset } = this.settings;
+    const { url, user, password, dataset, fusekiBase } = this.settings;
```

Comment:
This is also a BC for SemApps instances using secure datasets

Reply:
It's just an additional parameter that is only required for the new `triplestore.dataset.delete` action, so that should be fine?